metrics/config.go

```diff
@@ -58,9 +61,11 @@ const (
 
 	defaultBackendEnvName = "DEFAULT_METRICS_BACKEND"
 	defaultPrometheusPort = 9090
+	defaultPrometheusHost = ""
```

Comment:
do we need a const for ""? Or it has to be 0.0.0.0

Reply:
If we keep the convention of "" http will use `0.0.0.0` anyway, I guess I can remove the `const`.